tests/frontend/cli/conftest.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test fixtures and shared functions for strawberryfields.cli tests
+"""
+import pytest
+
+from strawberryfields import Program, ops
+from strawberryfields.api import Connection
+
+# pylint: disable=expression-not-assigned
+
+
+@pytest.fixture
+def prog():
+    """Program fixture."""
+    program = Program(2)
+    with program.context as q:
+        ops.Dgate(0.5) | q[0]
+    return program
+
+
+@pytest.fixture
+def connection():
+    """A mock connection object."""
```

Comment:
This looks like a real connection object 😆

Reply:
Oh, this is a really good catch, as it remained from a previous iteration of testing! As a matter of fact, these were not even used anymore in the test file.

For now, I've removed this file and moved the test file up a directory into `frontend`. Would you, however, suggest that indeed the `Mock` classes from the test file would go into a conftest file? (I can recreate this structure easily, if so :) )

Reply:
Nope, looks good how it is! My preference is always:

* Start off with fixtures/test data local/within the test file itself.

* If the fixtures/test data is too large, _or_ we identify that the fixtures can be used in other test files, we move them to conftest.